code/game/objects/items/toys.dm

```diff
@@ -415,6 +415,26 @@
 	playsound(user ? user : src, active ? 'sound/weapons/saberon.ogg' : 'sound/weapons/saberoff.ogg', 20, TRUE)
 	return COMPONENT_NO_DEFAULT_MESSAGE
 
+/obj/item/toy/sword/examine(mob/user)
+	. = ..()
+	. += span_notice("It has a lighting dial currently set to [saber_color] which looks like it can be turned with a <b>screwdriver</b>.")
```

Comment:
Add a contextual screentip too: [`/datum/element/contextual_screentip_tools`](https://github.com/tgstation/tgstation/blob/master/code/datums/elements/screentips/README.md#datumelementcontextual_screentip_tools)

Reply:
i dig it. will do when i'm not drunk.